bin/generate_report.py

```diff
@@ -377,6 +377,7 @@ def pdb_to_lddt(struct_files, generate_tsv):
     "colabfold": "ColabFold",
     "rosettafold_all_atom": "RosettaFold All-Atom",
     "helixfold3": "HelixFold3",
+    "rosettafold2na": "RoseTTAFold2NA",
```

Comment:
We should maybe either write `RosettaFold All-Atom` as `RoseTTaFold All-Atom` or change this for `RoseTTAFold2NA`, I will probably change the former but as you prefer